ex19.py

```diff
@@ -0,0 +1,12 @@
+import requests
+from bs4 import BeautifulSoup
+
+base_url = "http://www.vanityfair.com/society/2014/06/monica-lewinsky-humiliation-culture"
+r = requests.get(base_url)
+soup = BeautifulSoup(r.text,features="html.parser")
+
+all_p_cn_text_body = soup.select("p")
```

Comment:
Variable names should be proper. It should have some meaning & also, help the other developers to get the context & understand what data is stored in it.